Tests/Realm.Tests/Realm.Tests.csproj

```diff
@@ -109,7 +109,7 @@
         </None>
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(UseRealmNupkgsWithVersion)' == ''">
+    <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(UseRealmNupkgsWithVersion)' == '' AND '$(RealmTestsStandaloneExe)' == 'true'">
```

Comment:
Added this to prevent msbuild from complaining when we're building the iOS/tvOS tests without having built the macOS wrappers.